CHANGELOG.md

```diff
@@ -1,123 +1,130 @@
 #### Changes
-* Python: Added JSON.DEL JSON.FORGET commands  ([#1146](https://github.com/aws/glide-for-redis/pull/1146))
```

Comment:
As discussed, looks like there were a few changes accidentally included here, there should only be one extra line adding the notice about SINTER in this file